spec/grape/validations/params_scope_spec.rb

```diff
@@ -9,6 +9,58 @@ def app
     subject
   end
 
+  context 'setting a default' do
+    context 'propagates documentation' do
+      let(:documentation) { subject.routes.first.route_params }
+
+      it 'when the default value is truthy' do
+        subject.params do
+          optional :int, type: Integer, default: 42
+        end
+        subject.get '/' do
+        end
+        expect(documentation).to have_key('int')
+        expect(documentation['int']).to have_key(:default)
+        expect(documentation['int'][:default]).to eq(42)
+      end
```

Comment:
I relook this and `it 'when ...` definition seems unusual for me :)
maybe we can remove 'propagates documentation' context and split 'its' definitions to own contexts
```ruby
context 'when the default value is truthy' do
  before do
    subject.params do
      optional :int, type: Integer, default: 42
    end
    subject.get
  end
  it 'adds documentation to the value' do
   ...
  end
```
What do you think?